addons/web_kanban_sparkline/static/src/js/kanban_sparkline.js

```diff
@@ -31,6 +32,8 @@ instance.web_kanban.SparklineBarWidget = instance.web_kanban.AbstractField.exten
                 }, self.options);
             self.$el.sparkline(value, sparkline_options);
             self.$el.tooltip({delay: {show: self.options.delayIn || 0, hide: 0}, title: function(){return title}});
+            self.$el.css('height', self.$('canvas').height() + 2);
+            self.$el.css('width', self.$('canvas').width() + 2);
```

Comment:
He just tells me he asked you to do this. So my bad ^^"
Put it in a separated commit. Thanks !